app_feup/lib/redux/action_creators.dart

```diff
@@ -343,6 +344,7 @@ ThunkAction<AppState> getUserFees(Completer<Null> action) {
       store.dispatch(SetFeesLimitAction(feesLimit));
       store.dispatch(SetFeesStatusAction(RequestStatus.successful));
       store.dispatch(SetFeesRefreshTimeAction(currentTime));
+      store.dispatch(updateFeeNotifications(Completer(), feesLimit));
```

Comment:
Completers are only needed when you need to know when your action finishes, I don't think we need one here